Cargo.toml

```diff
@@ -7,9 +7,9 @@ rust-version = "1.73.0"
 
 [workspace.dependencies]
 connection-router = { version = "^0.1.0", path = "contracts/connection-router" }
-cosmwasm-std = "1.5.3"
-cosmwasm-schema = "1.3.3"
-cosmwasm-storage = "1.3.3"
+cosmwasm-std = "1.3.4"
+cosmwasm-schema = "1.3.4"
+cosmwasm-storage = "1.3.4"
```

Comment:
should we add a note here explaining we should be careful updating these dependencies and why?